### DISCUSSION TITLE
Can't find the Un-Discoverable plugin to make dummy Devices

### DISCUSSION THREAD

**Discussion by JackD-01** [netalertx/NetAlertX#749] [Q&A] [answered]
Hi, 

It is probably just me but I can not find the Un-Discoverable Plugin. I need to make 3 dummy plugins for my 3 inline power adapters who are not automatically picked up by any of the scanners. But I can't seem to find this plugin anywhere in NetAlertX. Is this no longer a function in the recent NetAlertX?. It was there in the old Pi-Alert and it still is there in the recent NetAlertX documentation. I cleaned all the folders and thereby all the old settings and did a new pull of the image and so a clean docker install with NetAlertX V24.7.18 Still can't find where to create dummy devices?
. I looked in settings and Integration/Plugins. 

Any advice would be appreciated......

(I am not a programmer, just a happy user of NetAlertX, so if I ask a stupid question please forgive me)

Regards, 
Jack

![NetAlertX - Settings](https://github.com/user-attachments/assets/6d3a7ea1-7e9d-42e1-ba63-dc7f36714b58)

**Comment by jokob-sk** [comment, marked as answer]
Hi @JackD-01 ,

Thanks for using NEtAlertX! You can read about how to load additional plugins, via the `LOADED_PLUGINS` setting, in the Plugins docs: https://github.com/jokob-sk/NetAlertX/tree/main/front/plugins#readme

Hope this helps!

**Reply by JackD-01** [reply]
Thanks for the advice. In Pi-Alert this plugin was loaded by default, in NetAlertX not, that is where I went wrong.
With you advice I was able to get the plugin loaded an got the ability to create 3 dummy devices for my inline power adapters.

Regards,
Jack
